src/TestFramework/Codeception/Config/Builder/MutationConfigBuilder.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Copyright © 2018 Tobias Stadler
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\TestFramework\Codeception\Config\Builder;
+
+use Infection\Mutant\Mutant;
+use Infection\TestFramework\Codeception\Config\MutationYamlConfiguration;
+use Infection\TestFramework\Config\MutationConfigBuilder as ConfigBuilder;
+
+class MutationConfigBuilder extends ConfigBuilder
+{
+    /**
+     * @var string
+     */
+    private $tempDirectory;
+
+    /**
+     * @var string
+     */
+    private $projectDirectory;
+
+    /**
+     * @var string
+     */
+    private $originalConfigPath;
+
+    public function __construct(string $tempDirectory, string $projectDirectory, string $originalConfigPath)
+    {
+        $this->tempDirectory = $tempDirectory;
+        $this->projectDirectory = $projectDirectory;
+        $this->originalConfigPath = $originalConfigPath;
+    }
+
+    public function build(Mutant $mutant): string
+    {
+        $_SERVER['CUSTOM_AUTOLOAD_FILE_PATH'] = $customAutoloadFilePath = sprintf(
```

Comment:
could you please expand on why do you need it?

Reply:
This is needed in order to autoload the mutated file. As far as I know Codeception does not support a custom bootstrap file (as e.g. PHPUnit)

Reply:
please use `putenv` instead of touching global variables

Reply:
Infection runs frameworks via symfony console which knows how to work with envs. We already use it :P.

p.s. IMHO Don't like to touch global variables

Reply:
I'm not sure to follow, I don't see how using the Console component helps to make it thread-safe here.

Maybe for reference: https://github.com/symfony/recipes/pull/156

Also I don't like to touch global variables much, but that's exactly what put/getenv are doings and regardless of what we are using we should use a prefix here IMO